src/aics-image-viewer/components/App/index.tsx

```diff
@@ -782,7 +689,14 @@ const App: React.FC<AppProps> = (props) => {
 
   // Rendering ////////////////////////////////////////////////////////////////
 
-  const showControls = { ...defaultShownControls, ...props.showControls };
+  const showControls = useMemo(
+    (): ShowControls => ({ ...defaultShownControls, ...props.showControls }),
+    [props.showControls]
+  );
```

Comment:
Why did this become a useMemo of a function?

Reply:
Discussed in a meeting, but so it's here: the profiler revealed that this prop was the only reason that some components continued to render, because this object was being regenerated on every render in order to apply defaults. The `useMemo` ensures we only make a new object when its value might actually change.